### ISSUE TITLE
[Bug] [a11y] Text contrast - consider increasing contrast ratio on "Add search engine" elements

### ISSUE THREAD

**Issue by softvision-miralobontiu** [mozilla-mobile/fenix#7724]
## Prerequisites
Make sure you have Google Accessibility Scanner installed.

## Steps to reproduce
1. Launch Fenix and go to Settings - Search - Add search engine.
2. Scan the page.

### Expected behavior
No major discrepancies suggested.

### Actual behavior
The "Replace query with "%s". Example:..." text is suggested to have a 3.00 ratio, instead of 1.24.

### Device information
* Android device: Samsung Galaxy Tab S3 (Android 8)
* Fenix version: RC 3.0.2, Beta build 3.1.0-beta 3, Nightly build from 01/15

![Screenshot_20200115-171340_Accessibility Scanner](https://user-images.githubusercontent.com/40694158/72445985-5b655100-37bb-11ea-866c-f40cb7dd4e44.jpg)

**Comment by sblatz** [comment]
Closing this as expected, disabled text does not need to meet high contrast standards.

**Comment by sv-sdeiac** [comment]
Hi, I investigated this issue and found that the contrast of the disabled text on the "Add search engine" is different from other disabled text areas (e.g. Delete browsing data on quit).

Devices:
- Google Pixel 3 (Android 10);
- Samsung Galaxy S9 (Android 8).

> Search engine disabled text
![Screenshot_20200408-114455](https://user-images.githubusercontent.com/40264165/78766305-c7f28500-7991-11ea-9ac8-6596de62a222.png)

> Delete browsing data on quit disabled text
![Screenshot_20200408-115129](https://user-images.githubusercontent.com/40264165/78766651-36cfde00-7992-11ea-9335-007a4f7bccd3.png)

Please consider increasing the contrast ratio on "Add search engine" elements to be the same as in the other parts, thanks.

**Comment by mcarare** [comment]
@asadotzler When this is disabled, the info in those elements have no importance. When selecting "Custom" (currently "Other") from the list and those elements are enabled the contrast ratio is 4.33. 
WCAG also mention this situation: 
> Text or images of text **that are part of an inactive user interface component**, that are pure decoration, that are not visible to anyone, or that are part of a picture that contains significant other visual content, **have no contrast requirement**.

Considering this, do you still think that we should change the color for disabled state? TY!

**Comment by liuche** [comment]
@mcarare we're going to prioritize this - is this something you're working on, or should someone eles pick it up? Since Asa prioritized this, we should do it.

**Comment by asadotzler** [comment]
Mihai is correct, these are disabled controls. I didn't realize that when I set the priority. This bug is invalid.
